versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/ContentAndState.java

```diff
@@ -36,11 +36,13 @@
 
   @Nonnull
   static <CONTENT> ContentAndState<CONTENT> of(
-      @Nonnull CONTENT refState, @Nonnull CONTENT globalState) {
-    return ImmutableContentAndState.<CONTENT>builder()
-        .refState(refState)
-        .globalState(globalState)
-        .build();
+      @Nonnull CONTENT refState, @Nullable CONTENT globalState) {
+    ImmutableContentAndState.Builder<CONTENT> b =
+        ImmutableContentAndState.<CONTENT>builder().refState(refState);
+    if (globalState != null) {
+      b.globalState(globalState);
```

Comment:
Would it be an error before this change?

Reply:
Simplified